libs/plugins/a-comp.lv2/a-comp#stereo.ttl.in

```diff
@@ -90,68 +97,88 @@
         lv2:default 0.000000 ;
         lv2:minimum 0.000000 ;
         lv2:maximum 45.000000 ;
-        unit:unit unit:db ;
+        unit:unit unit:db-display ;
     ] ,
-    [
+        [
         a lv2:OutputPort, lv2:ControlPort ;
         lv2:index 7 ;
+        lv2:name "Input Level" ;
+        lv2:symbol "inlevel" ;
+        lv2:default -45.000000 ;
+        lv2:minimum -45.000000 ;
+        lv2:maximum 0.000000 ;
+        unit:unit unit:db-display ;
+    ] ,
+    [
+        a lv2:OutputPort, lv2:ControlPort ;
+        lv2:index 8 ;
         lv2:name "Output Level" ;
         lv2:symbol "outlevel" ;
         lv2:default -45.000000 ;
         lv2:minimum -45.000000 ;
         lv2:maximum 0.000000 ;
-        unit:unit unit:db ;
+        unit:unit unit:db-display ;
     ] ,
     [
         a lv2:InputPort, lv2:ControlPort ;
-        lv2:index 8 ;
+        lv2:index 9 ;
         lv2:name "Sidechain" ;
         lv2:symbol "sidech" ;
-        lv2:default 0 ;
-        lv2:minimum 0 ;
-        lv2:maximum 1 ;
+        lv2:default 0.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 1.000000 ;
```

Comment:
Since the port is an integer, using int as min/default/max is preferred.  falktx' LV2 validator will prints warnings.